single_cell_analysis/scripts/00-data.download.processing/splatter.simulation.R

```diff
@@ -0,0 +1,121 @@
+# Qiwen Hu 2018
```

Comment:
Many lines in this file are quite long. Can you split long lines onto multiple lines?